### DISCUSSION TITLE
LFS Creates large files

### DISCUSSION THREAD

**Discussion by cheboot** [git-lfs/git-lfs#5205] [Q&A]
LFS Creates large files over in folder .git\lfs\objects so i can't even push it. Has anyone encountered a similar problem? How to solve it?

Uploading LFS objects: 100% (1772/1772), 17 GB | 7.6 MB/s, done.
[80cac8902c37291b439ca42f473be38b33482fa34bdce977a08d694115181ecd] Size must be less than or equal to 2147483648: [422] Size must be less than or equal to 2147483648
[962b74a6c37d091161ae29f16f4928aeaaa35fdbce7c3bf41f7cbee69313b334] Size must be less than or equal to 2147483648: [422] Size must be less than or equal to 2147483648
error: failed to push some refs to '

**Comment by bk2204** [comment]
Hey,

This message comes from the remote side (probably, in this case, GitHub).  One or more of the objects in your history are larger than 2 GiB, and GitHub limits the size of objects to 2 GiB for individuals (it's larger for teams and enterprises).

This isn't related to Git LFS the open-source client, which is merely reporting the error returned when the server rejects the item.  The client isn't creating large files which weren't in your working tree originally, so if you want to remove them from the history, you can rewrite your history such that they don't exist anymore, possibly using `git rebase -i`.
